Dockerfile

```diff
@@ -52,4 +54,6 @@ RUN chmod +x /usr/bin/enable-admin-container
 RUN groupadd -g 274 api
 RUN useradd -m -G users,api ssm-user
 
-CMD ["/usr/bin/amazon-ssm-agent"]
+ADD start_control_ssm.sh /usr/sbin/
+RUN chmod +x /usr/sbin/start_control_ssm.sh
```

Comment:
If you set the bits on the file that's added, then it comes in with the executable bits set - which avoids this extra command!

Reply:
Rather than change that in this PR, I've opened up an issue ([bottlerocket-os/bottlerocket/issues/1438](https://github.com/bottlerocket-os/bottlerocket/issues/1438)). I found a neat solution, but it requires Docker 20.10.